djtriggers/tests/test_models.py

```diff
@@ -67,6 +69,36 @@ def test_handle_execution_failure_tries_exceeded(self, mock_logger):
                                                                              try_count=original_tries+1)
         mock_logger.assert_called_with(trigger, message, level=WARNING)
 
+    @override_settings(DJTRIGGERS_TRIES_BEFORE_WARNING=3)
+    @override_settings(DJTRIGGERS_TRIES_BEFORE_ERROR=5)
+    @patch.object(TriggerLogger, 'log_message')
+    def test_handle_execution_failure_tries_limit(self, mock_logger):
```

Comment:
Is there a test that verifies that triggers aren't retried after they've maxed out their retries?

Reply:
I'll write one.